src/rules/no-unused-vars.md

```diff
@@ -4,24 +4,22 @@ layout: doc
 rule_type: problem
 ---
 
+在代码中声明了变量，但没有在任何地方使用，这很可能是由于不完整的重构造成的错误。这样的变量占用了代码中的空间，会导致读者的混淆。
 
+## 规则细节
 
-Variables that are declared and not used anywhere in the code are most likely an error due to incomplete refactoring. Such variables take up space in the code and can lead to confusion by readers.
+这个规则的目的是消除未使用的变量、函数和函数参数。
 
-## Rule Details
+一个变量 `foo` 被认为是被使用的，如果以下任何一项为真：
 
-This rule is aimed at eliminating unused variables, functions, and function parameters.
+* 它被调用（`foo()`）或构造（`new foo()`）
+* 它被读取（`var bar = foo`）
+* 它被作为参数传入一个函数 (`doSomething(foo)`)
+* 在一个函数中读取，并传递给另一个函数（`doSomething(function() { foo(); }）`）
```

Comment:
```suggestion
* 它被调用（`foo()`）或构造（`new foo()`）。
* 它被读取（`var bar = foo`）。
* 它被作为参数传入一个函数 (`doSomething(foo)`）。
* 在一个函数中读取，并传递给另一个函数（`doSomething(function() { foo(); }）`）。
```

In someplace full stop is there in other places it is not there. Can we make it consistent?

Reply:
I prefer to be consistent with the original, let me check